lib/rentlinx/default.rb

```diff
@@ -6,9 +6,11 @@ module Default
     USER_AGENT = "Rentlinx Ruby Client #{Rentlinx::VERSION}".freeze
 
     class << self
-      def headers
+      def headers(token: nil)
```

Comment:
Also, this addition breaks ruby 1.9 support :(. I like this syntax, but I think for a general library ruby 1.9 support should be maintained.